docs/en/ingest-management/release-notes/release-notes-8.7.asciidoc

```diff
@@ -105,6 +105,31 @@ To work around this problem, uninstall the {agent} and install it again with
 {fleet-server} enabled during the bootstrap process.
 ====
 
+--
+[[known-issue-2433-8-6-2-2]]
+.There is a known issue when upgrading Agents to 8.7.0 that are running Osquery.
+[%collapsible]
+====
+
+*Details*
+
+It is possible that Agents can get stuck in an "Updating" state. This is only known to occur if Agents have the Osquery Manager integration installed (#2433). Users can do the following work around the issue.
+
+
+For more information, refer to {agent-issue}2433[#2433].
+
+*Impact* +
+
+Wait for the 8.7.1 release to upgrade Agents to the 8.7.x line.
+
+Remove the Osquery Manager integration before upgrading. After the Agent is upgraded to 8.7.0, add the Osquery Manager integration back to the Agent.
+
+If you encounter this issue and Agents are stuck in "Updating", remove the Osquery Manager integration, upgrade, and then add it back.
+
+Note, you may need to use the Agent upgrade API in this scenario over the UI.
+Adds a known issue to the 8.7.0 release notes for this Agent upgrade issue: elastic/elastic-agent#2433
```

Comment:
General structure feedback:

```suggestion
*Details* +
{agent}s that have the Osquery Manager integration installed can get stuck in an "Updating" state. 
For more information, refer to {agent-issue}2433[#2433].

*Impact* +
Users can do the following work around the issue:

* Wait for the 8.7.1 release to upgrade {agent}s to the 8.7.x line.
* Remove the Osquery Manager integration before upgrading. After the {agent} has upgraded to 8.7.0, add the Osquery Manager integration back to the {agent}.
* If you encounter this issue and {agent}s are stuck in the "Updating" phase, remove the Osquery Manager integration, upgrade the {agent}, and then add it back.

NOTE: you may need to use the {agent} upgrade API in this scenario instead of the UI.
```